.eslintignore

```diff
@@ -1 +1,2 @@
 coverage/
+!.alexrc.js
```

Comment:
Shouldn't need to add this.

Reply:
Yeahhh I don’t love that it’s there but VSC was flagging a CJS export warning that was bugging me and ESLint ignores dotfiles by default

Reply:
Ah TIL it ignores dotfiles by default.